mock/py/mockbuild/plugins/mount.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, plugins, conf, buildroot):
         self.config = buildroot.config
         self.state = buildroot.state
         self.opts = conf
+        # Skip mounting user-specified mounts if we're in the boostrap chroot
+        if buildroot.is_bootstrap:
+            return
```

Comment:
This  needs to go one line lower. We need to enable this hook (to mount essential mounts), but we do not want to add custom mounts (in that for loop 2 lines below).

Reply:
Given that self.opts['dirs'] seems to only contain user defined (non-essential) mounts, and that the '_mountCreateDirs' function only deals with mounts listed there, it doesn't seem to have anything to do with essential mounts.

Also IMO its better not to run '_mountCreateDirs' if we're not going to mount so unnecessary mount points will not be created. 

IMO it would be kind of strange if this plugin that seems to only deal with user-defined mounts had something to do with the essential mounts...

Also in my testing the essential mount were being configure in the bootstrap chroot just fine...

Reply:
Right, I was confused.